integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/massindexing/MassIndexingConditionalExpressionsIT.java

```diff
@@ -29,12 +30,18 @@
 import org.junit.Rule;
 import org.junit.Test;
 
-/**
- * Test that the {@link MassIndexer} correctly indexes even complex entity hierarchies
- * where superclasses are indexed but not all of their subclasses, and vice-versa.
- */
 public class MassIndexingConditionalExpressionsIT {
 
+	// where T0 < T1 < T2
+	private static final Instant T0 = Instant.ofEpochMilli( 1_000_000 );
```

Comment:
Please follow Java conventions: single-letter names are reserved for type parameters. Maybe `INSTANT_0`? And `INT_0` below.